### PR TITLE
Fix status code for v2/getEntities when DB fails

### PR DESCRIPTION
**Pull request by Chandradeep-NEC**
Fix for Issue #3460

### CODE REVIEW

File: src/lib/serviceRoutinesV2/getEntities.cpp [hunk 1]
```diff
@@ -314,31 +314,23 @@ std::string getEntities
 
   // 03. Render Entities response
 
-  if (parseDataP->qcrs.res.contextElementResponseVector.size() == 0)
```
Comment (fgalan) [inline src/lib/serviceRoutinesV2/getEntities.cpp]: Not sure if the proposed change is correct...

It seems you have removed some checks (the one for `parseDataP->qcrs.res.contextElementResponseVector.size()` in the original external if clause and the one for `oe.code != SccNone` in the original internal if clause), I understand that with the aim of simpliying the logic.

However, although I don't remember why, I understand these checks are here for some reason :). In fact, if you look to the travis run of the functional tests at https://travis-ci.org/telefonicaid/fiware-orion/jobs/643774476?utm_medium=notification&utm_source=github_status you will see that some of the them, probably for this reason (did you run the test locally before creating the PR?)

Not sure, but maybe the issue should be solved in a lower part (in mongoBackend)

**Comment by fgalan** [comment]
Any update in the progress of this PR, please?

Thanks!

**Comment by fgalan** [comment]
This PR has opened comments that would need an answer, please. In addition, they have conflict that needs to be solved.

**Comment by ArqamFarooqui110719** [comment]
> This PR has opened comments that would need an answer, please. In addition, they have conflict that needs to be solved.

Hi @fgalan sir,
Sorry for late reply, I was busy in some other priority tasks.
I have looked into the issue, and currently `GET v2/entities` API is returning 500 Internal Server Error when DB fails. So I think this issue has been resolved. For reference I'm attaching screenshots:
**When DB is active:**

Screenshot (360)

**When DB is exited(fail):**

Screenshot (362)

If your understanding is also same then Please close this PR and issue #3460

**Comment by fgalan** [comment]
I agree with your analysis: the issue no longer occurs. Not sure when it happened, but probably some other issue fix has indirectly fixed this one.

Thus, I understand this PR is no longer needed so it will be closed. Thanks!

**Pull request closed without merging**